common/common-rest/src/main/java/org/apache/servicecomb/common/rest/AbstractRestInvocation.java

```diff
@@ -160,16 +163,24 @@ public void invoke() {
   }
 
   protected Response prepareInvoke() throws Throwable {
-    this.initProduceProcessor();
+    SCBStatus currentStatus = SCBEngine.getInstance().getStatus();
```

Comment:
why not just invoke ensureStatusUp, or refactor ensureStatusUp, so that no need to write the error message everywhere

Reply:
is it possible check in InvocationFactory.forProvider?
if need to send response, then catch exception for "forProvider"

Reply:
sounds good,Done